Assets/BackgroundParticles.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+public class BackgroundParticles : MonoBehaviour
+{
+    private ParticleSystem particles;
+
+    private void Start()
+    {
+        particles = GetComponent<ParticleSystem>();
+        particles.Play();
+    }
+
+    private void Update()
+    {
+        switch (GameManager.GameState)
+        {
+            case GameState.Start:
+                break;
+            case GameState.Play:
```

Comment:
Same as in other cases, simple if will work just fine since there are only two conditions.